tutorials/fundamentals/filter-key.md

```diff
@@ -0,0 +1,36 @@
+To filter an object using the keys, you can use JavaScript's `Object.hasOwnProperty()` method.
+
+```javascript
+const obj = { name: 'Masteringjs', location: 'Florida', help: true };
+let newObj = {}
+if(obj.hasOwnProperty('name')) {
+    newObj['name'] = obj['name'];
+}
+```
+
+## Object.keys()
+
+Another approach to filter using keys is to use the `Object.keys()` method.
+`Object.keys()` will return an array of the keys and you can then iterate through
```

Comment:
"`Object.keys` returns an array of keys, and then you can iterate through and filter accordingly." Remove some unnecessary words.